airbyte-ci/connectors/pipelines/pipelines/publish_pypi.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from textwrap import dedent
+from typing import List
+
+import anyio
+from dagger import ExecError
+from pipelines.bases import ConnectorReport, Step, StepResult, StepStatus
+from pipelines.contexts import PublishConnectorContext
+from pipelines.pipelines import metadata
+
+
+class PublishPyPIConnector(Step):
+    title = "Publish connector to PyPI"
+
+    async def _run(self) -> StepResult:
+        if self.context.connector.language not in ["python", "low-code"]:
+            return self.skip("Only Python connectors can be published to PyPI.")
+
+        try:
+            setup_cfg = dedent(
+                f"""
+                [metadata]
+                name = airbyte-{self.context.connector.technical_name}
+                version = {self.context.connector.version}
+                author = Airbyte
+                author_email = contact@airbyte.io
+            """
+            )
+
+            pypi_stdout = await (
+                self.context.dagger_client.container()
+                .from_("python:3.9-slim")
```

Comment:
Is is mandatory to use python:3.9 here or could we publish python 3.9 packages from a python 3.10 env?

Reply:
Would be team 3.10 if possible!